### PR TITLE
Learning Team Notes Oct 6 and Oct 13, 2016

### PR DESCRIPTION
**Pull request by toddjordan**
Thumbsup this comment to approve oct 6

### CODE REVIEW

File: docs-team/2016-10/october-06.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+LS: think so, not entirely sure.
+
+### Builds App
+
+RM: Deploys to S3 (my acct), waiting on permissions.
+
+### Module Unification
+
+RM: after talking to ben, yehuda, will have a page on the website learn page (during beta) for the perspective as a user (rfc geared toward implementers).
```
Comment (locks) [inline docs-team/2016-10/october-06.md]: dan, not ben ;P

**Comment by toddjordan** [comment]
thumbsup this comment to approve oct 13

File: docs-team/2016-10/october-13.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+RM: Would like to also run offline
+
+RM: I'll volunteer to talk to Mike about the issue to understand the problem more, also about the potential of getting a common hosted data service.  
+In general I'd like to review the guides from top to bottom.  
+We are mostly in agreement that we want to introduce mirage at some point, and we want things to work offline, but we do think there are things we can do to improve experience for brand new users
+
+### API App
+
+DB: Still working things out with Stanley _Update: DB up with him.  He's looking for help.  Looking to Todd to assist in remaining work._
```
Comment (acorncom) [inline docs-team/2016-10/october-13.md]: DB **met** up with him?

File: docs-team/2016-10/october-13.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+- TJ: Remove Mirage from Tutorial? https://github.com/emberjs/guides/issues/1695
+- DB: API repo transfer?
+- DB: Help wanted update
+- RM: [ES2015 primer](https://github.com/emberjs/guides/blob/es-primer/source/localizable/getting-started/es-primer.md)
+- RM+DB: design/marketing
+
+# Action Items 
+
+- [ ] RM: Check with CLI team re: are http-mocks deprecated / going away in the long-run?
```
Comment (nathanhammond) [inline docs-team/2016-10/october-13.md]: Nope.

Comment (nathanhammond) [inline docs-team/2016-10/october-13.md]: More color: it's supported, not deprecated, and we test it on every release. It's an unfortunately complex portion of Ember CLI which makes it desirable to remove support, but realistically in that case it should just be moved to a core-supported addon.

**Pull request merged**